### PR TITLE
Linked list

### PR DESCRIPTION
**Pull request by haitle16**
- [X] Top-level README “Table of Contents” is updated
 - [X] Feature tasks for this challenge are completed
 - [X] Unit tests written and passing
     - [X] “Happy Path” - Expected outcome
     - [X] Expected failure
     - [X] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [X] Summary, Description, Approach & Efficiency, Solution
     - [X] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: Data-Structures/linkedList/src/test/java/linkedList/LibraryTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package linkedList;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class LibraryTest {
+    @Test public void testSomeLibraryMethod() {
```
Comment (Zaffierce) [inline Data-Structures/linkedList/src/test/java/linkedList/LibraryTest.java:10]: Here you can do something like this so you don't have to instantiate so many linkedlists within your tests.
`LinkedList link = new LinkedList(); 
@before 
public void setUp() { 
  link.insert("Apples", 1); 
  link.insert("Bananas", 2); 
};`

and now, in each one of your tests, you can reference link. `@before` will run before each test and will create that new LinkedList for you, vice having to write the same thing a dozen times over.

Comment (haitle16) [reply Data-Structures/linkedList/src/test/java/linkedList/LibraryTest.java:10]: Thanks, will implement this idea.
